__tests__/tweet-routes.test.js

```diff
@@ -6,22 +6,31 @@ const connect = require('../lib/utils/connect');
 const mongoose = require('mongoose');
 const Tweet = require('../lib/models/tweet.js');
 
-
 describe('app routes', () => {
-    beforeAll(() => {
-      connect();
-    });
-  
-    beforeEach(() => {
-      return mongoose.connection.dropDatabase();
-    });
-  
-    afterAll(() => {
-      return mongoose.connection.close();
-    });
+  beforeAll(() => {
+    connect();
+  });
+  beforeEach(()=> {
+    return mongoose.connection.dropDatabase();
+  });
+  afterAll(()=> {
+    return mongoose.connection.close();
+  });
+
+  //test post route 
+  it('creates a comment', async() => {
+    const tweet = await Tweet.create({ handle: 'Black Phillip', text: 'Wouldst thou like to live deliciously?' });
 
-    it('gets a tweet by id', () => {
-        return Tweet.create({
-            
-        })
-    }
\ No newline at end of file
+    return request(app)
+      .post('/api/v1/comments/')
```

Comment:
you'll want to change this route to whatever you put in app.js (see below comment) so it might look like `.post('/api/v1/tweets')`